<?xml version="1.0"?>

<launch>
  <group ns="step_control_module">
    <param name="step_controller_plugin" value="thormang3_online_walking_plugin" />
  </group>
</launch>

